examples/simple-example/simple-example-core/src/main/java/io/miragon/miranum/integrations/example/application/port/in/SendMessageCommand.java

```diff
@@ -8,7 +10,8 @@
 @AllArgsConstructor
 @NoArgsConstructor
 public class SendMessageCommand extends TenantAwareCommand {
-
-    private String name;
+    @ElementTemplateProperty(type = PropertyType.STRING, label = "content", editable = true)
```

Comment:
The editable property should maybe set to true by default. Not sure if it not is already the case, if so => can be omitted